src/components/DashboardHeader/DashboardHeader.tsx

```diff
@@ -8,14 +8,27 @@ import { Icon } from '../Icon/Icon';
 import * as classNames from 'classnames';
 import './DashboardHeader.scss';
 
+function debounce(func, wait, immediate?) {

```

Comment:
lodash already has a debounce function. https://lodash.com/docs/4.17.4#debounce. better to use that?